### ISSUE TITLE
How can we record and save file as .wav ?

### ISSUE THREAD

**Issue by krixs3112** [hyochan/react-native-audio-recorder-player#592]
I'm reaching out to request a new feature for the project. We are currently working on a use case where we need to record audio and save the file in the .wav format.(Android)

Add a new option or method to the recording functionality that allows users to specify the desired output format, including .wav.
Ensure that the recorded audio is saved in the specified format, maintaining the quality and characteristics of the original recording.

**Comment by sravanvudem** [comment]
same requirement for me tooo
any patch or workaround highly appriciated

**Comment by NazarenkoAnna2021** [comment]
find it in another issue, maybe it can help:

const PATH = Platform.select({
    ios: 'audio.VAW',
    android: `${cacheDir}/audio.VAW`,
});

audioRecorderPlayer.startRecorder(PATH);

**Comment by davidburson** [comment]
I found on Android I can save the file with a .wav extension by including it in the path that is passed to startRecorder, but the file is not actually saved in WAVE format.  Is saving in WAVE format possible on Android yet?

**Comment by dipankar4765** [comment]
Hi @davidburson did you get any update or workaround for this?

**Comment by davidburson** [comment]
@dipankar4765 no, for wav on Android I'm using [react-native-audio-record](https://github.com/goodatlas/react-native-audio-record).  It's archived, but it is still working for this case for me.

**Comment by gauravCelloip** [comment]
Steps:

Navigate to the following path in your project:
node_modules/react-native-audio-recorder-player/src/java/RNAudioPlayerModuleKit.kt

Go to line 420 and change the file format from mp4 to wav.

Save the file and rerun the project to apply the changes.

**Comment by hyochan** [comment]
Thanks for the report. To triage this I need:

- [ ] `react-native-nitro-sound` version (from `package.json`)
- [ ] `react-native-nitro-modules` version
- [ ] React Native version
- [ ] Platform + OS version (iOS or Android, exact version)
- [ ] A minimal reproduction (CodeSandbox / github repo / code snippet)
- [ ] Exact steps to reproduce
- [ ] Full error log / stack trace (not a screenshot)

Without these I can't move this forward — I'll auto-close issues with no follow-up in 14 days.

> 🤖 AI-native maintenance. See #787 for context.

**Comment by hyochan** [comment]
Duplicate of #419. Closing in favor of the original thread. Please continue discussion there.
